### PR TITLE
Pull in changes for nimbus

### PR DESCRIPTION
**Pull request by claytonv**
Pull over all changes required to MPM/XSAPI/Profile for the Nimbus Project

### CODE REVIEW

File: Include/xsapi/multiplayer.h [hunk 1]
```diff
@@ -2248,6 +2248,11 @@ class multiplayer_session_member
     /// </summary>
     _XSAPIIMP const string_t& team_id() const;
 
+	/// <summary>
```
Comment (rakhad) [inline Include/xsapi/multiplayer.h]: >	 [](start = 0, length = 1)

tabs to spaces, here and everywhere else

File: Include/xsapi/matchmaking.h [hunk 2]
```diff
@@ -239,7 +239,8 @@ class matchmaking_service
         _In_ const string_t& hopperName,
         _In_ const std::chrono::seconds& ticketTimeout,
         _In_ preserve_session_mode preserveSession,
-        _In_ const web::json::value& ticketAttributesJson = web::json::value()
+        _In_ const web::json::value& ticketAttributesJson = web::json::value(),
+        _In_ bool isSymmetric = false
```
Comment (rakhad) [inline Include/xsapi/matchmaking.h]: >_In_ bool isSymmetric = false [](start = 8, length = 29)

Add a comment for this parameter

File: Source/Services/Multiplayer/Manager/WinRT/MultiplayerGameSession_WinRT.h [hunk 3]
```diff
@@ -25,6 +25,14 @@ public ref class MultiplayerGameSession sealed
     /// </summary>
     DEFINE_PTR_PROP_GET_STR_OBJ(CorrelationId, correlation_id);
 
+
+    DEFINE_PTR_PROP_GET_STR_OBJ(ServerConnectionString, matchmaking_server_connection_string);
```
Comment (rakhad) [inline Source/Services/Multiplayer/Manager/WinRT/MultiplayerGameSession_WinRT.h]: >DEFINE_PTR_PROP_GET_STR_OBJ(ServerConnectionString, matchmaking_server_connection_string); [](start = 4, length = 90)

Needs summary comment

Comment (rakhad) [reply Source/Services/Multiplayer/Manager/WinRT/MultiplayerGameSession_WinRT.h]: Here and below

---
In reply to: [252432890](https://github.com/Microsoft/xbox-live-api/pull/518#discussion_r252432890) [](ancestors = 252432890)

**Comment by rakhad** [comment]
>         /// <remarks>Calls V2 GET /users/batch/profile/settings</remarks>

V3

---
Refers to: Include/xsapi/profile.h:154 in f783298. [](commit_id = f78329853d0a4fa022c0484decae9582c366d9f1, deletion_comment = False)

File: Source/Services/Social/profile_service.cpp [hunk 4]
```diff
@@ -81,7 +129,7 @@ profile_service::get_user_profiles(
         _T("/users/batch/profile/settings"),
         xbox_live_api::get_user_profiles
         );
-    httpCall->set_xbox_contract_version_header_value(_T("2"));
+    httpCall->set_xbox_contract_version_header_value(_T("3"));
```
Comment (rakhad) [inline Source/Services/Social/profile_service.cpp]: >httpCall->set_xbox_contract_version_header_value(_T("3")); [](start = 4, length = 58)

Does this result in a breaking change?

**Comment by rakhad** [comment]
>         /// <remarks>Calls V2 GET /users/batch/profile/settings</remarks>

V3

---
Refers to: Include/xsapi/profile.h:141 in f783298. [](commit_id = f78329853d0a4fa022c0484decae9582c366d9f1, deletion_comment = False)

File: Include/xsapi/multiplayer.h (hunk 1 above)
Comment (claytonv) [reply Include/xsapi/multiplayer.h]: We should include this in the Contributing.md

File: Source/Services/Social/profile_service.cpp (hunk 4 above)
Comment (claytonv) [reply Source/Services/Social/profile_service.cpp]: It shouldn't. Just adds more fields

**Pull request merged**